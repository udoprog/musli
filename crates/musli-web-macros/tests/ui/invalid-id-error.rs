musli_web_macros::define! {
    #[musli(id = 0)]
    type Type1;

    #[musli(id = 32768)]
    type Type2;
}

fn main() {}