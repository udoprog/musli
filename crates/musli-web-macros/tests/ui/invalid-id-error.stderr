error: Message id `0` not in range 1-32767
 --> tests/ui/invalid-id-error.rs:2:18
  |
2 |     #[musli(id = 0)]
  |                  ^

error: Message id `32768` not in range 1-32767
 --> tests/ui/invalid-id-error.rs:5:18
  |
5 |     #[musli(id = 32768)]
  |                  ^^^^^
